app/services/cfe/submission_error.rb

```diff
@@ -0,0 +1,10 @@
+module CFE
+  class SubmissionError < StandardError
+    attr_reader :http_status
+
```

Comment:
I'd suggest using [Nesty](https://github.com/skorks/nesty) here to preserve the original data error in the new error.